src/routes.js

```diff
@@ -161,6 +161,10 @@ The following meta fields are supported for bottom-level routes:
 
   - standalone (default: false): If standalone is `true` then application layout
     elements like navigation bar, background color, etc are not rendered.
+
+  - skipAutoLogout (default: false): If `true`, no alert will be displayed when
```

Comment:
What do you think about moving this closer to the descriptions of the route meta fields related to login? They feel thematically related. They could all go in one section: section about login could be renamed to "Login/logout" or "Session". Or `skipAutoLogout` could go in its own new section named "Logout" that comes after the section named "Login". No preference from my end on the exact section titles, the main thing I think is that it'd be nice for these route meta fields to be close by each other.